home/migrations/0066_alter_whatsapptemplate_category.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.2.11 on 2024-03-22 09:02
```

Comment:
Can we squash all these migrations into one before we merge? (And maybe do it occasionally along the way to avoid unnecessary bloat?)

Reply:
Cool, squashed.  Believe it or not, I have squashed it a few times already but forgot to do it before this push.  Thx